GruntFile.js

```diff
@@ -57,7 +58,7 @@ module.exports = function(grunt) {
         bump: {
             options: {
                 files: bumpFiles,
-                commitFiles: bumpFiles,
+                commitFiles: bumpFiles.concat(['src/ChangeLog.html']),
```

Comment:
this is nice for when you `grunt bump-only:x` update the change log and then `grunt bump-commit`.